### DISCUSSION TITLE
Cascading esm/cjs formats when importing from other libraries

### DISCUSSION THREAD

**Discussion by mcshaz** [egoist/tsup#816] [Q&A]
What an awesome tool - thank you very much.

I have a typescript file which imports
```typescript
import _zonedTimeToUtc from 'date-fns-tz/esm/zonedTimeToUtc'
```
when I build with `tsup src/index.ts --format esm,cjs` I would like the ESM version to use the exact same import statement (which is working perfectly). 

However, I would like the commonJS version to use the .cjs exported by the `date-fns-tz` library. That is, the .cjs output will ideally be `var zonedTimeToUtc = require( 'date-fns-tz/zonedTimeToUtc')` (with '/esm' removed)
is this possible, and if so, how? If not are there any solutions or other libraries you would suggest? - thank you.

**Comment by klujanrosas** [comment]
@mcshaz Did you figure this one out by any chance?
